adapter/memQuota/util/util.go

```diff
@@ -120,7 +120,7 @@ func (qu *QuotaUtil) ReapDedup() {
 	qu.OldDedup = qu.RecentDedup
 	qu.RecentDedup = t
 
-	if qu.Logger.VerbosityLevel(4) {
+	if len(t) > 0 && qu.Logger.VerbosityLevel(4) {
 		qu.Logger.Infof("Running repear to reclaim %d old deduplication entries", len(t))
```

Comment:
let's also fix: "repear" -> "reaper"

Reply:
How do we keep ending up with this change being removed? This is at least the third time.

Reply:
This PR can now remove the entire adapter, but I will send a separate PR for that.